test/test-network/conf/25-sysctl.network

```diff
@@ -13,3 +13,4 @@ IPv6ProxyNDP=yes
 IPv6AcceptRA=no
 IPv4AcceptLocal=yes
 IPv4ReversePathFilter=no
+MulticastIGMPVersion=no
```

Comment:
`no`, which maps to `0`, seems to be the kernel's default. To check if the setting is really applied, please use one of v1, v2, or v3.

Reply:
done, but please double check this because it is not clear for me exactly how the test works